account_renumber/test/create_moves.py

```diff
@@ -19,30 +18,21 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
-
-"""
-Script that creates large amounts of account moves on different days,
-that can be used later for testing the renumber wizard.
-"""
```

Comment:
I am.  In Python modules, the first docstring will be used as `__doc__` for the module, so this is more than just a habit. Just tested with the last pep8 version and didn't had a warning for that.